


Team retreat at the villa Finaly, Florence, Italy</div>


<h2>Former Members</h2>
<em>Left in 2022:</em>
<ul>
 	<li><span style="color: #008c9e;">Thibault Rolland</span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Clément Mantoux</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Virgilio Kmetzsch</span> - Former PhD student. Now research engineer at Owkin</li>
 	<li><span style="color: #008c9e;">Johann Faouzi</span> - Former PhD student &amp; Postdoctoral fellow. Now lecturer at ENSAI Rennes</li>
 	<li><span style="color: #008c9e;">Dario Saracino</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Simona Bottani</span> - Former PhD student. Now Postdoctoral fellow at Helmholtz Zentrum, Munich, Germany</li>
</ul>
<em>Left in 2021:</em>
<ul>
 	<li><span style="color: #008c9e;">Elina Thibeau-Sutre</span> - Former PhD student. Now Postdoctoral fellow at Twente University, Enschede, Netherland</li>
 	<li><span style="color: #008c9e;">Federica Cacciamani</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Arnaud Valladier</span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Paul Vernhet</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Tiziana Cattai</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Raphaël Couronné</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Stéphane Epelbaum</span> - Former Faculty</li>
</ul>
<em>Left in 2020:</em>
<ul>
 	<li><span style="color: #008c9e;">Alexandre Bône</span> - Former PhD student. Now Augmented Intelligence Specialist at Guerbet</li>
 	<li><span style="color: #008c9e;">Manon Ansart</span> - Former PhD student. Now Postdoctoral fellow at the LIO laboratory, hosted at ETS, Montreal</li>
 	<li><span style="color: #008c9e;">Arnaud Marcoux</span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Adam Wild</span> - Former Engineer</li>
 	<li><span style="color: #008c9e;">Vincent Henry</span> - Former Postdoctoral fellow</li>
 	<li><span style="color: #008c9e;">Emmannuel Mauduit</span>  - <a href="http://www.linkedin.com/in/emmanuelle-mauduit-539aa93a">LinkedIn</a> - Former coordinator the ICM center of  Neuroinformatics</li>
 	<li><span style="color: #008c9e;">Thomas Lartigue</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Giulia Bassignana</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Alexandre Routier</span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Wen Wei</span> - Former PhD student</li>
</ul>
&nbsp;

<em>Left in 2019 :</em>
<ul>
 	<li><span style="color: #008c9e;">Maxime Louis</span> - Former PhD Student, now at Pixyl</li>
 	<li><span style="color: #008c9e;">Pascal Lu</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Benoit Martin</span> - Former engineer. Now PhD student</li>
 	<li><span style="color: #008c9e;">Alexis Guyot</span> - Former Postdoctoral fellow</li>
 	<li><span style="color: #008c9e;">Junhao Wen</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Fanny Grosselin</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Jorge Samper-González</span> - Former PhD student &amp; Postdoctoral fellow, now data scientist at Qynapse</li>
</ul>
&nbsp;

<em>Left in 2018 and before:</em>
<ul>
 	<li><span style="color: #008c9e;">Catalina Obando</span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Jeremy Guillon</span> - Former PhD student - <a href="http://guillonjeremy.co">webpage</a></li>
 	<li><span style="color: #008c9e;">Michael Bacci</span> - Former engineer, now at EURONEXT</li>
 	<li><span style="color: #008c9e;">Sabrina Fontanella</span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Clementine Fourrier</span> - Former engineer, now at SED INRIA</li>
 	<li><span style="color: #008c9e;">Federico Battiston</span> - Former visiting researcher</li>
 	<li><span style="color: #008c9e;">Takoua Kaaouana</span> - Former postdoctoral fellow</li>
 	<li><span style="color: #008c9e;">Xavier Navarro</span> - Former postdoctoral fellow, now researcher at <a href="http://melomind.com/fr/" target="_blank" rel="noopener noreferrer">MyBrainTechnologies</a></li>
 	<li><span style="color: #008c9e;">Romain Valabrègue</span> - Former Inserm Research Engineer</li>
 	<li><span style="color: #008c9e;">Eric Bardinet</span> - Former CNRS Research Engineer - <a href="http://cenir.org/equipe-irm/38-eric-b0" target="_blank" rel="noopener noreferrer">webpage</a></li>
 	<li><span style="color: #008c9e;">Sara Fernandez-Vidal </span>- Former Inserm Research Engineer - <a href="http://bebg.chups.jussieu.fr/members/sara-fernandez-vidal/">webpage</a></li>
 	<li><span style="color: #008c9e;">Vera Dinkelacker </span>- Former associated clinician</li>
 	<li><span style="color: #008c9e;">Thomas Estienne </span>-  Former administrative assistant</li>
 	<li><span style="color: #008c9e;">Chabha Azouani </span>- Former clinical research associate</li>
 	<li><span style="color: #008c9e;">Kelly Martineau </span> - Former clinical research associate</li>
 	<li><span style="color: #008c9e;">Sonia Djobeir </span> - Former clinical research associate</li>
 	<li><span style="color: #008c9e;">Hugo Dary </span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Ludovic Fillon </span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Mathieu Dubois </span> - Former engineer</li>
 	<li><span style="color: #008c9e;">Barbara Gris </span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Géraldine Rousseau </span>-Former PhD student</li>
 	<li><span style="color: #008c9e;">Marika Rudler </span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Jean-Baptiste Schiratti </span> - Former PhD student</li>
 	<li><span style="color: #008c9e;">Pietro Gori </span> - Former PhD student - Now Assistant Professor, Telecom ParisTech - <a href="https://perso.telecom-paristech.fr/pgori/">webpage</a></li>
 	<li><span style="color: #008c9e;">Ana Fouquier </span> - Former postdoctoral fellow - Now R&amp;D Engineer, Cardiawave Company</li>
 	<li><span style="color: #008c9e;">Soledad Fernandez-Garcia </span> - Former postdoctoral fellow - Now Assistant Professor, University of Sevilla</li>
 	<li><span style="color: #008c9e;">Tristan Moreau </span> - Former postdoctoral fellow - Now Postdoctoral fellow at ICM Dubois/Lévy\'s team</li>
 	<li><span style="color: #008c9e;">Susovan Pal</span> - Former Postdoctoral fellow - Now Postdoctoral fellow at UCLA (University of California at Los Angeles)</li>
 	<li><span style="color: #008c9e;">François Touvet </span> - Former engineer - Now Pipeline TD Junior at Ellipse Studio</li>
 	<li><span style="color: #008c9e;">Alexis Mocellin </span>- Former engineer</li>
 	<li><span style="color: #008c9e;">Claire Cury </span>- Former PhD student - Now Postdoctoral fellow at UCL (University College London)</li>
 	<li><span style="color: #008c9e;">Johanne Germain </span>- Former clinical research associate -  Now Clinical Research Associate in Toulouse</li>
 	<li><span style="color: #008c9e;">Ali Bouyahia </span>- Former clinical research associate - Now Psychiatrist<img class="size-medium wp-image-997 alignright" src="https://www.aramislab.fr/wp-content/uploads/2014/02/IMG_20160916_180329-300x215.jpg" alt="" width="300" height="215" /></li>
 	<li><span style="color: #008c9e;">Linda Marrakchi-Kacem </span>- Former Postdoctoral fellow - Now Assistant Professor, Tunisia</li>
 	<li><span style="color: #008c9e;">Xavier Badé </span>- Clinical Research Associate - Now Clinical Research Assistant at Umanis</li>
 	<li><span style="color: #008c9e;">Yohan Attal </span>- Postdoctoral fellow - Now CEO of <a href="http://melomind.com/fr/" target="_blank" rel="noopener noreferrer">MyBrainTechnologies</a></li>
 	<li><span style="color: #008c9e;">Mario Chavez </span>- CNRS Researcher (CR1)</li>
 	<li><span style="color: #008c9e;">Claude Adam </span>- Neurologist (PH), AP-HP</li>
 	<li><span style="color: #008c9e;">Sophie Dupont </span>- Professor of Neurology (PU-PH), UPMC / AP-HP</li>
 	<li><span style="color: #008c9e;">Damien Galanaud </span>- Professor of Neuroradiology (PU-PH), UPMC / AP-HP</li>
 	<li><span style="color: #008c9e;">Dominique Hasboun </span>- Associate Professor of Neuroanatomy (MCU), UPMC</li>
 	<li><span style="color: #008c9e;">Yves Samson </span>- Professor of Neurology (PU-PH), UPMC / AP-HP</li>
 	<li><span style="color: #008c9e;">Lionel Thivard </span>- Neurologist (PH), AP-HP</li>

<h2>Former Members</h2>
<ul>
 	<li><span style="color: #008c9e;">Romain Valabrègue </span>- Inserm Research Engineer - <a href="http://www.cenir.org/equipe-irm/37-romain.html">webpage</a> - romain.valabregue@upmc.fr</li>
 	<li><span style="color: #008c9e;">Eric Bardinet </span>- CNRS Research Engineer - <a href="http://cenir.org/equipe-irm/38-eric-b0" target="_blank">webpage</a> - eric.bardinet@upmc.fr</li>
 	<li><span style="color: #008c9e;">Sara Fernandez-Vidal </span>- Inserm Research Engineer - <a href="http://bebg.chups.jussieu.fr/members/sara-fernandez-vidal/">webpage</a> - sara.fernandez_vidal@upmc.fr</li>
 	<li><span style="color: #008c9e;">Vera Dinkelacker </span>- Rothschild Foundation - v.dinkelacker@gmail.com</li>
 	<li><span style="color: #008c9e;">Thomas Estienne </span>- UPMC, CATI project - thomas.estienne@icm-institute.org</li>
 	<li><span style="color: #008c9e;">Martina Sundqvist </span>- Ecole Normale Supérieure de Paris</li>
 	<li><span style="color: #008c9e;">Thomas Jacquemont </span>- Ecole Normale Supérieure de Paris</li>
 	<li><span style="color: #008c9e;">Aurore Bussalb </span>- Institut Supérieur de BioSciences de Paris</li>
 	<li><span style="color: #008c9e;">Carlos Tor Diez </span>- Télécom ParisTech</li>
 	<li><span style="color: #008c9e;">Ayoub Louati </span>- Ecole Polytechnique de Tunisie</li>
 	<li><span style="color: #008c9e;">Paul Jusselin </span>- Ecole Normale Supérieure de Cachan</li>
 	<li><span style="color: #008c9e;">Chabha Azouani </span>- CATI project - chabha.azouani@upmc.fr</li>
 	<li><span style="color: #008c9e;">Kelly Martineau </span>- CATI project – kelly.martineau@icm-institute.org</li>
 	<li><span style="color: #008c9e;">Sonia Djobeir </span>- CATI project – soniadjobeir@gmail.com</li>
 	<li><span style="color: #008c9e;">Hugo Dary </span>- CATI project - dary.hugo@gmail.com</li>
 	<li><span style="color: #008c9e;">Ludovic Fillon </span>- CATI project - ludovic.fillon@upmc.fr</li>
 	<li><span style="color: #008c9e;">Mathieu Dubois </span>- CATI project - mathieu.dubois@icm-institute.org</li>
 	<li><span style="color: #008c9e;">Barbara Gris </span>- gris@cmla.ens-cachan.fr</li>
 	<li><span style="color: #008c9e;">Géraldine Rousseau </span>- geraldine.rouseau@psl.ap-hop-paris.fr</li>
 	<li><span style="color: #008c9e;">Marika Rudler </span>- marika.rudler@psl.aphp.fr</li>
 	<li><span style="color: #008c9e;">Jean-Baptiste Schiratti </span>- jean-baptiste.schiratti@cmap.polytechnique.fr</li>
 	<li><span style="color: #008c9e;">Pietro Gori </span>- Postdoctoral fellow - Now Assistant Professor, Telecom ParisTech</li>
 	<li><span style="color: #008c9e;">Ana Fouquier </span>- - Postdoctoral fellow - Now R&amp;D Engineer, Cardiawave Company</li>
 	<li><span style="color: #008c9e;">Soledad Fernandez-Garcia </span> - Postdoctoral fellow - Now Assistant Professor, University of Sevilla</li>
 	<li><span style="color: #008c9e;">Tristan Moreau </span> - Postdoctoral fellow - Now Postdoctoral fellow at ICM Dubois/Lévy\'s team</li>
 	<li><span style="color: #008c9e;">Alexandre Pron </span> - Master\'s student (Université Paris Descartes) - Now PhD student at INT, Marseille</li>
 	<li><span style="color: #008c9e;">Susovan Pal</span> - Postdoctoral fellow - Now Postdoctoral fellow at UCLA (University of California at Los Angeles)</li>
 	<li><span style="color: #008c9e;">François Touvet </span> - Engineer - Now Pipeline TD Junior at Ellipse Studio</li>
 	<li><span style="color: #008c9e;">François Deloche </span> - Intern (École Polytechnique)</li>
 	<li><span style="color: #008c9e;">Alexis Mocellin </span>- Intern (Ecole Polytechnique)</li>
 	<li><span style="color: #008c9e;">Claire Cury </span>- PhD student - Now Postdoctoral fellow at UCL (University College London)</li>
 	<li><span style="color: #008c9e;">Johanne Germain </span>- Clinical Research Associate - Now Clinical Research Associate in Toulouse</li>
 	<li><span style="color: #008c9e;">Ali Bouyahia </span>- Clinical Research Associate - Now Psychiatrist</li>
 	<li><span style="color: #008c9e;">Linda Marrakchi-Kacem </span>- Postdoctoral fellow - Now Assistant Professor, Tunisia</li>
 	<li><span style="color: #008c9e;">Kanza Dekkiche </span>- Master\'s student</li>
 	<li><span style="color: #008c9e;">Antoine Latrille </span>- Master\'s student (ESIEE Paris) - Now Ingénieur Développeur Web at Easyvoyage</li>
 	<li><span style="color: #008c9e;">Guillaume Ruffin </span>- Intern</li>
 	<li><span style="color: #008c9e;">Evgeny Zuenko </span>- Master\'s student</li>
 	<li><span style="color: #008c9e;">Xavier Badé </span>- Clinical Research Associate - Now Clinical Research Assistant at Umanis</li>
 	<li><span style="color: #008c9e;">Maxime Corduant </span>- Master\'s student</li>
 	<li><span style="color: #008c9e;">Kevin Roussel </span>- Master\'s student</li>
 	<li><span style="color: #008c9e;">Yohan Attal </span>- Postdoctoral fellow - Now CEO of <a href="http://melomind.com/fr/" target="_blank">MyBrainTechnologies</a></li>
 	<li><span style="color: #008c9e;">Mario Chavez </span>- CNRS Researcher (CR1) - mario.chavez@upmc.fr</li>
 	<li><span style="color: #008c9e;">Claude Adam </span>- Neurologist (PH), AP-HP - claude.adam@psl.aphp.fr</li>
 	<li><span style="color: #008c9e;">Sophie Dupont </span>- Professor of Neurology (PU-PH), UPMC / AP-HP - sophie.dupont@psl.aphp.fr</li>
 	<li><span style="color: #008c9e;">Damien Galanaud </span>- Professor of Neuroradiology (PU-PH), UPMC / AP-HP - damien.galanaud@icm-institute.org</li>
 	<li><span style="color: #008c9e;">Dominique Hasboun </span>- Associate Professor of Neuroanatomy (MCU), UPMC - dominique.hasboun@upmc.fr</li>
 	<li><span style="color: #008c9e;">Yves Samson </span>- Professor of Neurology (PU-PH), UPMC / AP-HP - yves.samson@psl.aphp.fr</li>
 	<li><span style="color: #008c9e;">Lionel Thivard </span>- Neurologist (PH), AP-HP - lionel.thivard@psl.aphp.fr</li>


<h2>Direction<img class=" wp-image-986 alignleft" src="https://www.aramislab.fr/wp-content/uploads/2014/02/Inria-0230-182-1-300x259.jpg" alt="" width="242" height="209" /></h2>
     <ul>
          <li><a href="https://www.aramislab.fr/perso/colliot/index.html" target="_blank"><span style="color: #008c9e;">Olivier Colliot </span></a>- CNRS Research Director (DR) - <a href="https://www.aramislab.fr/perso/colliot/index.html" target="_blank">webpage</a> - olivier.colliot@upmc.fr</li>
          <li><a href="https://who.rocq.inria.fr/Stanley.Durrleman/index.html" target="_blank">Stanley Durrleman </a>- Inria Researcher - <a href="https://who.rocq.inria.fr/Stanley.Durrleman/index.html" target="_blank">webpage</a> - stanley.durrleman@inria.fr</li>
     </ul>
     

     <h2>Researchers</h2>
     <ul>
          <li><a href="https://www.aramislab.fr/perso/colliot/index.html" target="_blank"><span style="color: #008c9e;">Olivier Colliot </span></a>- CNRS Research Director (DR) - <a href="https://www.aramislab.fr/perso/colliot/index.html" target="_blank">webpage</a> - olivier.colliot@upmc.fr</li>
          <li><span style="color: #008c9e;">Fabrizio De Vico Fallani </span>- Inria Researcher (HDR) - <a href="https://sites.google.com/site/fabriziodevicofallani/" target="_blank">webpage</a> - fabrizio.devicofallani@gmail.com</li>
          <li><a href="https://who.rocq.inria.fr/Stanley.Durrleman/index.html" target="_blank">Stanley Durrleman </a>- Inria Researcher - <a href="https://who.rocq.inria.fr/Stanley.Durrleman/index.html" target="_blank">webpage</a> - stanley.durrleman@inria.fr</li>
     </ul>

     <h2>Medical Faculty / Clinicians</h2>
     <ul>
          <li><span style="color: #008c9e;">Anne Bertrand </span>- Neuroradiologist (PH), AP-HP/Inria - anne.bertrand@psl.aphp.fr</li>
          <li><span style="color: #008c9e;">Didier Dormont </span>- Professor of Neuroradiology (PU-PH), UPMC / AP-HP - didier.dormont@psl.aphp.fr</li>
          <li><span style="color: #008c9e;">Stéphane Epelbaum </span>- Neurologist (PH), AP-HP - stephane.epelbaum@aphp.fr</li>
     </ul>
     <h2>Postdocs</h2>
     <ul>
          <li><span style="color: #008c9e;">Federico Battiston </span></li>
          <li><span style="color: #008c9e;">Ninon Burgos </span></li>
          <li><span style="color: #008c9e;">Marie-Constance Corsi </span></li>
          <li><span style="color: #008c9e;">Takoua Kaaouana </span> - takoua.kaaouana@upmc.fr</li>
          <li><span style="color: #008c9e;">Xavier Navarro </span> - x.navarro.s@gmail.com</li>
     </ul>
     
     <h2>PhD students</h2>
     <ul>
          <li><span style="color: #008c9e;">Manon Ansart </span></li>
          <li><span style="color: #008c9e;">Alexandre Bône </span></li>
          <li><span style="color: #008c9e;">Fanny Grosselin </span>- grosselin.fanny@gmail.com</li>
          <li><span style="color: #008c9e;">Jérémy Guillon </span>- jeremy.guillon@inria.fr</li>
          <li><span style="color: #008c9e;">Igor Koval </span></li>
          <li><span style="color: #008c9e;">Maxime Louis </span></li>
          <li><span style="color: #008c9e;">Pascal Lu </span></li>
          <li><span style="color: #008c9e;">Catalina Obando Forero </span>- catalina.obando@inria.fr</li>
          <li><span style="color: #008c9e;">Alexandre Routier </span>- alexandre.routier@upmc.fr</li>
          <li><span style="color: #008c9e;">Jorge Samper Gonzales </span>- jsampergonzalez@gmail.com</li>
          <li><span style="color: #008c9e;">Wen Wei </span></li>
          <li><span style="color: #008c9e;">Junhao Wen </span>- junhao.wen@inria.fr</li>
     </ul>

     <h2>Engineers</h2>
     <ul>
          <li><span style="color: #008c9e;">Michael Bacci </span></li>
          <li><span style="color: #008c9e;">Simona Bottani </span></li>
          <li><span style="color: #008c9e;">Sabrina Fontanella </span></li>
          <li><span style="color: #008c9e;">Clémentine Fourrier </span></li>
          <li><span style="color: #008c9e;">Arnaud Marcoux </span></li>
     </ul>

     <h2>Master students</h2>
<ul>
 	<li><span style="color: #008c9e;">Tiziana Cattai </span></li>
 	<li><span style="color: #008c9e;">Giulia Bassignana </span></li>
 	<li><span style="color: #008c9e;">Camille Chrétien </span></li>
 	<li><span style="color: #008c9e;">Hugo Boniface </span></li>
 	<li><span style="color: #008c9e;">Jungying Fang </span></li>
</ul>

<h2>Administrative assistants</h2>
<ul>
 	<li><span style="color: #008c9e;">Hélène Milome </span>- Inria</li>
 	<li><span style="color: #008c9e;">Corinne Omer </span>- CNRS - corinne.omer@upmc.fr</li>
</ul>


<h2>Associated engineers of the neuroimaging core facility</h2>
<ul>
 	<li><span style="color: #008c9e;">Laurent Hugueville </span>- CNRS Engineer - <a href="http://www.cenir.org/lequipe-meg-eeg.html">webpage</a> - laurent.hugueville@upmc.fr</li>
 	<li><span style="color: #008c9e;">Denis Schwartz </span>- Inserm Research Engineer - <a href="http://www.cenir.org/lequipe-meg-eeg.html">webpage</a> - Denis.Schwartz@upmc.fr</li>
</ul>

<h2>Associated collaborators</h2>
<ul>
 	<li><span style="color: #008c9e;">Stéphanie Allassonnière </span>- École Polytechnique - <a href="https://sites.google.com/site/stephanieallassonniere/" target="_blank">webpage</a> - stephanie.allassonniere@polytechnique.edu</li>
 	<li><span style="color: #008c9e;">Marie Chupin </span>- CNRS Research Engineer (IR) - marie.chupin@upmc.fr</li>
</ul>





<h2><strong>Context and general aim</strong></h2>
<p style="text-align: justify;">Multiple characteristics of brain diseases can now be measured in living patients thanks to the tremendous progress of neuroimaging, genomic and biomarker technologies. Collection of multimodal data in large patient databases provide a comprehensive view of brain alterations, biological processes, genetic risk factors and symptoms.</p>
<p style="text-align: justify;"><strong>The general aim of the team is to build numerical models of brain diseases from multimodal patient data</strong> based on the development of specific data-driven approaches. To this end, we develop new data representations and statistical learning approaches that can integrate multiple types of data acquired in the living patient including neuroimaging, peripheral biomarkers, clinical and omics data (genetics, transcriptomics...). In particular, we develop methods to highlight networks of interactions among multiple sources of data, to track data changes during disease progression, and to automatically predict current or future clinical outcomes from these data.</p>
<p style="text-align: justify;">We apply these models to neurodegenerative disease (Alzheimer\'s disease and other dementia, Multiple Sclerosis, Parkinson\'s disease...). They shall allow to deepen our understanding of neurological diseases and to develop new decision support systems for diagnosis, prognosis and design of clinical trials.</p>

<strong><span style="color: #333333; font-family: \'Pontano Sans\',arial,sans-serif; font-size: 20px; line-height: normal;">New representations from multimodal medical images</span></strong>
<p style="text-align: justify;">Combining multiple neuroimaging modalities is necessary to obtain a comprehensive picture of alterations in brain diseases (atrophy, anatomical disconnections, functional connectivity alterations, metabolic alterations, abnormal protein deposits…). Such a combination is a non-trivial task because different types of information are conveyed by the different modalities, which in turns leads to different natural data representations (meshes and curves for geometrical information, signals, networks). We propose to build new integrated data representations from multiple modalities. Such representations will be subsequently entered into statistical models and decision support systems. Specifically, we will introduce representations that can integrate geometrical information (anatomical surfaces extracted from anatomical MRI, white matter tracts extracted from diffusion MRI) together with functional (PET, ASL, EEG/MEG) and microstructural information.</p>
&nbsp;


<strong><span style="color: #333333; font-family: \'Pontano Sans\',arial,sans-serif; font-size: 20px; line-height: normal;">Network theoretic approaches to integrate heterogeneous brain networks </span></strong>
<p style="text-align: justify;">The complexity of biological systems often emerges from interactions between components at multiple spatial and temporal scales. Neglecting this information, and analyzing separately the levels of such scales, is an oversimplification of the real phenomenon. We propose a methodological framework that aims, on the one hand, to integrate information from networks describing different modes of connectivity (e.g. multi-modal, cross-frequency) and, on the other hand, to statistically model the organizational mechanisms of temporally dynamic networks (e.g. nonstationary, longitudinal). Target applications include: i) human learning in brain-computer interface, ii) prediction of neurodegenerative disease progression, and iii) identification of driving nodes in biological pathways (gene expression networks).</p>
&nbsp;
<p style="text-align: justify;"><strong><span style="color: #333333; font-family: \'Pontano Sans\',arial,sans-serif; font-size: 20px; line-height: normal;">Spatio-temporal models to build trajectories of disease progression from longitudinal data</span></strong>
Longitudinal data sets are collected to capture variable temporal phenomena, which may be due to ageing or disease progression for instance. They consist in the observation of several individuals, each of them being observed at multiple points in time. The statistical exploitation of such data sets is notably difficult since data of each individual follow a different trajectory of changes and at its own pace. This difficulty is further increased if observations take the form of structured data like images or measurements distributed at the nodes of a mesh, and if the measurements themselves are normalized data or positive definite matrices for which usual linear operations are not defined. Our team has contributed to the definition of a generic theoretical and algorithmic framework for learning typical trajectories from longitudinal data sets. This framework is built on tools from the Riemannian geometry to describe trajectories of changes for any kind of data and their variability within a group both in terms of the direction of the trajectories and the pace at which trajectories are followed. The inference is based on a stochastic Expectation Maximization (EM) algorithm coupled with Markov Chain Monte Carlo methods.</p>
&nbsp;

<strong><span style="color: #333333; font-family: \'Pontano Sans\',arial,sans-serif; font-size: 20px; line-height: normal;">Decision support systems for diagnosis, prognosis and design of clinical trials</span></strong>
<p style="text-align: justify;">Based on the new representations and statistical models, we design decision support systems for diagnosis, prognosis and design of clinical trials. These systems are based on: i) personalization of statistical models to predict evolution at the patient level; ii) new machine learning approches for classification and regression on high-dimensional and structured data; iii) content-based retrieval techniques.</p>
&nbsp;



<h2><strong>External collaborations</strong></h2>
<h4>Methodological collaborations</h4>
<ul>
 	<li>University College London, <a href="http://www.ucl.ac.uk/cmic/homepage" target="_blank">Center for Medical Image Computing</a> (<a href="http://cmic.cs.ucl.ac.uk/staff/sebastien_ourselin/" target="_blank">Sébastien Ourselin</a>, <a href="http://www0.cs.ucl.ac.uk/staff/D.Alexander/" target="_blank">Daniel Alexander</a>)</li>
 	<li>Scientific Computing and Imaging (<a href="http://www.sci.utah.edu/" target="_blank">SCI</a>) Institute, University of Utah, USA (<a href="http://www.sci.utah.edu/~gerig/" target="_blank">Guido Gerig</a>, <a href="https://www.bioen.utah.edu/directory/profile.php?userID=252" target="_blank">Sarang Joshi</a>, <a href="http://www.sci.utah.edu/~prastawa/" target="_blank">Marcel Prastawa</a>)</li>
 	<li>Departement of Physics. <a href="http://ph.qmul.ac.uk/" target="_blank">Queen Mary</a> <a href="http://ph.qmul.ac.uk/" target="_blank">University of London</a>, UK (<a href="http://www.maths.qmul.ac.uk/~latora/" target="_blank">Vito Latora</a>)</li>
 	<li><a href="https://www.cmrr.umn.edu/" target="_blank">Center for Magnetic Resonance Research</a>, University of Minnesota, USA (<a href="http://www.cmrr.umn.edu/facultystaff/pfvdm.shtml" target="_blank">Pierre-François Van de Moortele</a>, Tom Henry, <a href="http://www.neurosci.umn.edu/faculty/ugurbil.html" target="_blank">Kamil Ugurbil</a>)</li>
 	<li>Inria <a href="https://team.inria.fr/asclepios/" target="_blank">Asclepios</a> (<a href="http://www-sop.inria.fr/members/Nicholas.Ayache/ayache.html" target="_blank">Nicholas Ayache</a>)</li>
 	<li><a href="http://www.ens-cachan.fr/" target="_blank">ENS de Cachan</a> (<a href="http://atrouve.perso.math.cnrs.fr/" target="_blank">Alain Trouvé</a>)</li>
 	<li><a href="http://www.parisdescartes.fr/" target="_blank">Université Paris-Descartes</a> (<a href="http://www.mi.parisdescartes.fr/~glaunes/" target="_blank">Joan Glaunès</a>)</li>
 	<li><a href="http://www.anthropobiologie.fr/index.php?clear=1" target="_blank">Laboratoire AMIS</a>, Université Paul Sabatier, Toulouse (<a href="http://www.anthropobiologie.fr/content.php?itemId=6&amp;id=6" target="_blank">José Braga</a>, <a href="http://www.anthropobiologie.fr/content.php?itemId=115&amp;id=6&amp;cur=0&amp;date&amp;manager&amp;categPath&amp;" target="_blank">Jean Dumoncel</a>)</li>
 	<li><a href="https://www.pasteur.fr/en" target="_blank">Institut Pasteur</a>, Paris (<a href="http://www.ura2182.cnrs-bellevue.fr/rtoro/" target="_blank">Roberto Toro</a>)</li>
 	<li><a href="https://www.insead.edu/" target="_blank">INSEAD</a>, Fontainebleau (<a href="https://www.insead.edu/faculty-research/faculty/theodoros-evgeniou" target="_blank">Theodoros Evgeniou</a>)</li>
 	<li>CEA <a href="http://www-dsv.cea.fr/dsv/instituts/institut-d-imagerie-biomedicale-i2bm/services/neurospin-neurospin" target="_blank">Neurospin</a> (<a href="http://www.lnao.fr/spip.php?rubrique7" target="_blank">Jean-François Mangin</a>, Alexandre Vignaud, <a href="http://www.eu-aims.eu/the-group/consortium/neurospin-cea/" target="_blank">Vincent Frouin</a>, <a href="http://www-dsv.cea.fr/dsv/instituts/institut-d-imagerie-biomedicale-i2bm/services/neurospin-neurospin/recherche/unites-de-recherche/unite-de-recherche-en-neuroimagerie-applicative-clinique-et-translationnelle-uniact" target="_blank">Lucie Hertz-Pannier</a>)</li>
</ul>
<h4>Medical collaborations</h4>
<ul>
 	<li><a href="http://www.ch-sainte-anne.fr/" target="_blank">Sainte-Anne Hospital</a>, Paris (<a href="http://scholar.google.fr/citations?user=CBzHpxEAAAAJ&amp;hl=fr" target="_blank">Catherine Oppenheim</a>, Marie Sarazin)</li>
 	<li><a href="http://www.chu-bordeaux.fr/" target="_blank">Bordeaux University Hospital</a> (<a href="http://cervco.aphp.fr/equipe/CDufouil.en.htm" target="_blank">Carole Dufouil</a>)</li>
 	<li><a href="http://www.cyceron.fr/" target="_blank">Cycéron</a>, Caen University Hospital (<a href="http://www.u1077.caen.inserm.fr/multimodal-imaging-of-brain-pathologies/?lang=en" target="_blank">Gaël Chételat</a>, <a href="http://www.u1077.caen.inserm.fr/" target="_blank">Francis Eustache</a>, Béatrice Desgranges)</li>
 	<li><a href="http://www.chru-lille.fr/" target="_blank">Lille University Hospital </a>(Christine Delmaire)</li>
</ul>

 
 <h2><strong>Local collaborations</strong></h2>
<h4>Methodological collaborations</h4>
<ul>
 	<li><a href="http://www.cenir.org/fr/equipe-irm.html" target="_blank">CENIR MRI</a> core facility (<a href="http://www.cenir.org/equipe-irm/7-stephane-l.html" target="_blank">Stéphane Lehéricy</a>, Eric Bardinet, Romain Valabrègue)</li>
 	<li><a href="http://www.cenir.org/fr/lequipe-meg-eeg.html" target="_blank">CENIR MEG/EEG</a> core facility (Nathalie George, Denis Schwartz, Laurent Hugueville)</li>
 	<li>ICM/IHU Bioinformatics/biostatistics core facility (Ivan Moszer)</li>
 	<li><a href="http://www.imed.jussieu.fr/en/" target="_blank">Laboratoire d\'Imagerie Biomédicale</a> (Marie-Odile Habert)</li>
</ul>
<h4>Medical collaborations</h4>
<ul>
 	<li>IM2A / ICM <a href="http://research.icm-institute.org/team/35/homepage" target="_blank">Bruno Dubois\'s team</a> (Bruno Dubois, Harald Hampel, Marc Teichmann, Isabelle Le Ber)</li>
 	<li>ICM <a href="http://research.icm-institute.org/team/30/homepage" target="_blank">Alexis Brice\'s team</a> (Alexis Brice, Isabelle Le Ber, Christel Depienne, Jean-Christophe Corvol)</li>
 	<li>ICM <a href="http://conam.synology.me/marie/Home.html" target="_blank">Marie Vidailhet / Stéphane Lehéricy\'s team</a> (Marie Vidailhet, Stéphane Lehéricy, Andreas Hartmann, Yulia Worbe)</li>
 	<li>ICM <a href="http://icm-institute.org/en/team/team-lubetzki-stankoff/" target="_blank">Catherine Lubetzki / Bruno Stankoff\'s team</a> (Bruno Stankoff, Benedetta Bodini)</li>
 	<li>ICM <a href="http://icm-institute.org/en/team/team-baron-nait-oumesmar/" target="_blank">Brahim Nait Oumesmar / Anne Baron Van Evercooren\'s team</a> (Violetta Zujovic)</li>
 	<li>Department of Neuroradiology, <a href="http://pitiesalpetriere.aphp.fr/" target="_blank">Pitié-Salpêtrière Hospital</a></li>
</ul>



     <strong><span>Network theoretic approaches to integrate heterogeneous brain networks </span></strong>
     The complexity of biological systems often emerges from interactions between components at multiple spatial and temporal scales. 
     Neglecting this information, and analyzing separately the levels of such scales, is an oversimplification of the real phenomenon. 
     We propose a methodological framework that aims, on the one hand, to integrate information from networks describing different modes 
     of connectivity (e.g. multi-modal, cross-frequency) and, on the other hand, to statistically model the organizational mechanisms of 
     temporally dynamic networks (e.g. nonstationary, longitudinal). Target applications include: i) human learning in brain-computer interface, 
     ii) prediction of neurodegenerative disease progression, and iii) identification of driving nodes in biological pathways (gene expression 
     networks).</p>
     &nbsp;
     
     <strong><span></span>Spatio-temporal models to build trajectories of disease progression from longitudinal data</span></strong>
     Longitudinal data sets are collected to capture variable temporal phenomena, which may be due to ageing or disease progression for 
     instance. They consist in the observation of several individuals, each of them being observed at multiple points in time. The statistical 
     exploitation of such data sets is notably difficult since data of each individual follow a different trajectory of changes and at its own 
     pace. This difficulty is further increased if observations take the form of structured data like images or measurements distributed at the 
     nodes of a mesh, and if the measurements themselves are normalized data or positive definite matrices for which usual linear operations are
      not defined. Our team has contributed to the definition of a generic theoretical and algorithmic framework for learning typical trajectories 
      from longitudinal data sets. This framework is built on tools from the Riemannian geometry to describe trajectories of changes for any kind of 
      data and their variability within a group both in terms of the direction of the trajectories and the pace at which trajectories are followed. 
      The inference is based on a stochastic Expectation Maximization (EM) algorithm coupled with Markov Chain Monte Carlo methods.</p>
     &nbsp;


    

     If you are interested in joining the team (with respect to any of our research topics), do not hesitate to send us an e-mail stating your field of interest and a CV. We may have other open positions than those advertised below. </strong>You can directly contact one of the <a href="https://www.aramislab.fr/?page_id=4" target="_blank" rel="noopener noreferrer">permanent researchers</a> of the team, or the <a href="mailto:olivier.colliot@upmc.fr">head</a> of the team for a general inquiry. Our team is pluridisciplinary and we welcome applicants with different profiles and expertises (computer science, electrical engineering, neuroimaging, neuroscience, medicine...).

     <h1><span style="font-size: 20px;">Postdocs / Scientists
    </span></h1>
    <ul>
         <li><a href="https://www.aramislab.fr/files/job_offers/JobOffer_ARAMIS.pdf">Statistical Learning in Medical Imaging</a> - <span style="text-decoration: underline;">Duration:</span> from 2 to 5 years - <span style="text-decoration: underline;">Starting date:</span>  anytime - <span style="text-decoration: underline;">Contact:</span> stanley.durrleman@inria.fr</li>
         <li><a href="https://www.aramislab.fr/wp-content/uploads/2016/10/postdoc_HIPLAY7.pdf">Medical image computing for high-resolution 7 Tesla MRI</a> - <span style="text-decoration: underline;">Duration:</span> from 2 to 3 years - <span style="text-decoration: underline;">Starting date:</span>  as soon as possible - <span style="text-decoration: underline;">Contact:</span> olivier.colliot@upmc.fr</li>
         <li><a href="https://www.aramislab.fr/wp-content/uploads/2017/02/Post-doc-2017_IPL_Neuromarkers_For_diffusion-2.pdf">Knowledge models for analysis and interpretation of genetic data</a> - <span style="text-decoration: underline;">Duration:</span> 18 months - <span style="text-decoration: underline;">Starting date:</span>  November 2017 - <span style="text-decoration: underline;">Contact:</span> olivier.colliot@upmc.fr</li>
         <li><del><a href="https://www.aramislab.fr/wp-content/uploads/2014/02/Research_position_NETBCI.pdf" target="_blank" rel="noopener noreferrer">Dynamic brain networks in brain-computer interface learning</a> - <span style="text-decoration: underline;">Duration</span>: from 2 to 3 years - <span style="text-decoration: underline;">Starting date</span>: 1 April 2016 - <span style="text-decoration: underline;">Contact:</span> fabrizio.devicofallani@gmail.com</del></li>
    </ul>

    <h1><span style="font-size: 20px;">PhD thesis
    </span></h1>
    <ul>
         <li><del><a href="https://www.aramislab.fr/files/job_offers/PhD_Inria_ICM.pdf">Learning methods for the spatiotemporal analysis of longitudinal data - Applications to the diagnosis, prognosis and monitoring of Alzheimer’s disease</a>  -  <span style="text-decoration: underline;">Starting date</span>: September 2016 (maybe combined with an internship in Spring 2016) - <span style="text-decoration: underline;">Contact:</span> stanley.durrleman@inria.f</del>r</li>
    </ul>

    <h2>Engineers / Software developers</h2>
<ul>
 	<li><a href="https://www.aramislab.fr/files/job-offers/Software_Dev.pdf">Software Development Engineer</a> - Scientific Computing and High Performance Computing in Medical Imaging - <span style="text-decoration: underline;">Starting date</span>: December 2017</li>
 	<li><a href="https://www.aramislab.fr/wp-content/uploads/2016/12/software_developer-2.pdf">Software Development Engineer</a>- Modeling progression of brain diseases from neuroimaging data - <span style="text-decoration: underline;">Starting date</span>: as soon as possible</li>
 	<li><del><a href="https://www.aramislab.fr/wp-content/uploads/2016/05/software_developper.pdf">Software Developer</a> - Development of software for analysis of multimodal medical imaging data - <span style="text-decoration: underline;">Starting date</span>: as soon as possible</del></li>
 	<li><del><a href="https://www.aramislab.fr/wp-content/uploads/2016/05/data_manager_image_analyst.pdf">Data Manager / Image Analyst</a> - Modeling progression of Alzheimer’s disease from brain imaging data - Starting date: as soon as possible</del></li>
 	<li><del><a href="https://www.aramislab.fr/files/job_offers/SoftwareEngineer_Inria_ICM.pdf">Development of a software product for the construction of virtual models of brain disease progression </a>- <span style="text-decoration: underline;">Duration:</span> from 2 to 4 years - <span style="text-decoration: underline;">Starting date</span>: as soon as possible - <span style="text-decoration: underline;">Contact:</span> stanley.durrleman@inria.fr</del></li>
</ul>

<h2> Master Internships / <em>Stages de Master</em></h2>
<ul>
 	<li>See PhD thesis above</li>
</ul>


<a href="https://www.aramislab.fr/files/job_offers/MicroscopicImaging.pdf">Analysis of 3D microscopic brain images at high resolution</a> - Starting date: September 2018 (maybe combined with an internship in Spring 2018) - <span style="text-decoration: underline;">Contact:</span> stanley.durrleman@inria.fr



Clinica 
Clinica is a software platform for multimodal brain image analysis in clinical research studies. It makes it easy to apply advanced analysis tools to large scale clinical studies. For that purpose, it integrates a comprehensive set of processing tools for the main neuroimaging modalities: currently MRI (anatomical, functional, diffusion) and PET, in the future, EEG/MEG.
For each modality, Clinica allows to easily extract various types of features (regional measures, parametric maps, surfaces, curves, networks). Such features are then subsequently used as input of machine learning, statistical modeling, morphometry or network analysis methods. </span>

<u>References</u>
<ul>
 	<li>Routier, A., Guillon, J., Burgos, N., Samper-Gonzalez, J., Wen, J., Fontanella, S., Bottani, S., Jacquemont, T., Marcoux, A., Gori, P., Lu, P., Moreau, T., Bacci, M., Durrleman, S., Colliot, O. <em> Clinica: an open source software platform for reproducible clinical neuroscience studies</em>. In <strong><em><u>OHBM </u></em></strong> 2018. <a href="https://hal.inria.fr/hal-01760658/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Routier A, Marcoux A, Diaz Melo M, Guillon J, Samper-González J, Wen J, Bottani S, Guyot A, Thibeau-Sutre E, Teichmann M, Habert M-O, Durrleman S, Burgos N, Colliot O: <em>New Advances in the Clinica Software Platform for Clinical Neuroimaging Studies</em>. In <span style="text-decoration: underline;"><em><strong>OHBM</strong></em></span> 2019. <a href="https://hal.inria.fr/hal-02132147/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Routier A, Marcoux A, Diaz Melo M, Samper-González J, Wild A, Guyot A, Wen J, Thibeau- Sutre E, Bottani S, Durrleman S, Burgos N, Colliot O: <em>New Longitudinal and Deep Learning Pipelines in the Clinica Software Platform</em>. In <span style="text-decoration: underline;"><em><strong>OHBM</strong></em></span>, 2020. <a href="https://hal.inria.fr/hal-02549242/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Samper-González J, Burgos N, Bottani S, Fontanella S, Lu P, Marcoux A, Routier A, Guillon J, Bacci M, Wen J, Bertrand A, Bertin H, Habert MO, Durrleman S, Evgeniou T, Colliot O. <em>Reproducible evaluation of classification methods in Alzheimer\'s disease: Framework and application to MRI and PET data</em>. <strong><em><u>Neuroimage</u></em></strong><em>, </em>183: 504–521, 2018. <a href="https://hal.inria.fr/hal-01858384/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Wen J, Thibeau-Sutre E, Samper-González J, Routier A, Bottani S, Durrleman S, Burgos N, Colliot O: <em>Convolutional Neural Networks for Classification of Alzheimer’s Disease: Overview and Reproducible Evaluation</em>, <span style="text-decoration: underline;"><em><strong>Medical Image Analysis</strong></em></span>, 63: 101694, 2020 <a href="https://hal.inria.fr/hal-02562504/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Marcoux A, Burgos N, Bertrand A, Teichmann M, Routier A, Wen J, Samper-González J, Bottani S, Durrleman S, Habert M-O, Colliot O: <em>An Automated Pipeline for the Analysis of PET Data on the Cortical Surface</em>. <strong><span style="text-decoration: underline;"><em>Frontiers in Neuroinformatics</em></span></strong>, 12, 2018. <a href="https://www.frontiersin.org/articles/10.3389/fninf.2018.00094/pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
<div class="col-md-6">[su_button url="http://www.clinica.run/" size="5" style="flat" background="#702082" color="white" center="yes"]Website[/su_button]</div>
<div class="col-md-6">[su_button url="mailto:olivier.colliot(a)inria(dot)fr" size="5" style="flat" background="#702082" color="white" center="yes"]Contact[/su_button]</div>

ClinicaDL 
ClinicaDL is an open-source deep learning software for reproducible neuroimaging processing. This library was developed from the <a href="https://github.com/aramis-lab/AD-DL">AD-DL project</a>. The combination of ClinicaDL and Clinica allows performing an end-to-end neuroimaging analysis, from the download of raw data sets to the interpretation of trained networks, including neuroimaging preprocessing, quality check, label definition, architecture search, and network training and evaluation.
We implemented ClinicaDL to bring answers to three common issues encountered by deep learning users who are not always familiar with neuroimaging data: (1) the format and preprocessing of neuroimaging data sets, (2) the contamination of the evaluation procedure by data leakage and (3) a lack of reproducibility.
</span>
<u>References</u>
<ul>
 	<li>Elina Thibeau-Sutre, Mauricio Diaz, Ravi Hassanaly, Alexandre Routier, Didier Dormont, Olivier Colliot, Ninon Burgos: <em>ClinicaDL: an open-source deep learning software for reproducible neuroimaging processing, <strong>HAL Preprint</strong></em>, 2021. <a href="https://www.frontiersin.org/articles/10.3389/fninf.2018.00094/pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /></a> <a href="https://www.frontiersin.org/articles/10.3389/fninf.2018.00094/pdf" target="_blank" rel="noopener noreferrer">Paper in PDF</a></li>
</ul>
git 
mail 
site 


Deformetrica
Deformetrica is a software for the statistical analysis of 2D and 3D shape data. It essentially computes deformations of the 2D or 3D ambient space, which, in turn, warp any object embedded in this space, whether this object is a curve, a surface, a structured or unstructured set of points, or any combination of them.</span></div>

<ul>
    <li>S. Durrleman, S., Prastawa, M., Charon, N., Korenberg, J.R., Joshi, S., Gerig, G., Trouvé, A. <em> Morphometry of anatomical shape complexes with dense deformations and sparse parameters.</em>. In <strong><em><u>Neuroimage</u></em></strong> 101(1): 35-49, 2014 <a href="http://www.sciencedirect.com/science/article/pii/S1053811914005205" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
    <li>Bône, A., Louis, M., Martin, B., &amp; Durrleman, S. <em> Deformetrica 4: an open-source software for statistical shape analysis.</em> In <strong><em><u>nternational Workshop on Shape in Medical Imaging</u></em></strong> Springer, Cham, 2018. p. 3-13. <a href="https://hal.inria.fr/hal-01874752/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>

<div class="col-md-6">[su_button url="http://www.deformetrica.org/" size="5" style="flat" background="#702082" color="white" center="yes"]Website[/su_button]</div>
<div class="col-md-6">[su_button url="mailto:stanley.durrleman(a)inria(dot)fr" size="5" style="flat" background="#702082" color="white" center="yes"]Contact[/su_button]</div>


<h2>Brain network toolbox</h2>
A list of MATLAB routines for characterizing brain network topology though graph theoretical indices can be found at the website of the FreeBorN consortium, which promotes the interaction and visibility of the research teams studying brain connectivity and network theory.</span>&nbsp;
logo : /uploads/2014/09/Logo_alto_100.png

A list of MATLAB routines for characterizing brain network topology though graph theoretical indices can be found at the website of the FreeBorN consortium, which promotes the interaction and visibility of the research teams studying brain connectivity and network theory.</span>&nbsp;
<div class="row">
<div class="col-md-6">[su_button url="https://sites.google.com/site/fr2eborn/download" size="5" style="flat" background="#702082" color="white" center="yes"]Website[/su_button]</div>
<div class="col-md-6">[su_button url="mailto:fabrizio.devicofallani(a)gmail(dot)com" size="5" style="flat" background="#702082" color="white" center="yes"]Contact[/su_button]</div>
</div>

<h2>Leasp</h2>
<div class="row">
<div class="col-md-3">
</div>
<div class="col-md-8">
<span class="softdescription">
    Leaspy is a software package for the statistical analysis of <strong>longitudinal data</strong>, particularly <strong>medical</strong> data that comes in a form of <strong>repeated observations</strong> of patients at different time-points. Considering these series of short-term data, the software aims at :
    <ul dir="auto">
        <li>recombining them to reconstruct the long-term spatio-temporal trajectory of evolution</li>
        <li>positioning each patient observations relatively to the group-average timeline, in term of both temporal differences (time shift and acceleration factor) and spatial differences (different sequences of events, spatial pattern of progression, ...)</li>
        <li>quantifying impact of cofactors (gender, genetic mutation, environmental factors, ...) on the evolution of the signal</li>
        <li>imputing missing values</li>
        <li>predicting future observations</li>
        <li>simulating virtual patients to un-bias the initial cohort or mimics its characteristics</li>
   </ul>
   <p dir="auto" data-sourcepos="55:1-57:252">The software package can be used with scalar multivariate data whose progression can be modelled by a logistic shape, an exponential decay or a linear progression. The simplest type of data handled by the software are scalar data: they correspond to one (univariate) or multiple (multivariate) measurement(s) per patient observation. This includes, for instance, clinical scores, cognitive assessments, physiological measurements (e.g. blood markers, radioactive markers) but also imaging-derived data that are rescaled, for instance, between 0 and 1 to describe a logistic progression.</p>

   <u>References</u>
   <ul>
        <li>I. Koval, A. Bone, M. Louis, S. Bottani, A. Marcoux, J. Samper-Gonzalez, N. Burgos, B. Charlier, A. Bertrand, S. Epelbaum, O. Colliot, S. Allassonniere &amp; S. Durrleman, <em>Intensive application for Alzheimer\'s Disease progression: AD Course Map charts Alzheimer\'s disease progression</em>, <em><strong><u>Scientific Reports</u></strong></em><u>,</u> 2021. <a href="http://www.jmlr.org/papers/volume18/17-197/17-197.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /></a> <a href="https://hal.inria.fr/hal-01964821v3/document">Paper in PDF</a></li>
        <li>Schiratti J-B, Allassonniere S, Colliot O, Durrleman S.<em>A Bayesian mixed-efects model to learn trajectories of changes from repeated manifold-valued observations</em>. In <strong><em><u>Journal of Machine Learning Research (JMLR)</u></em></strong> 18(1), 4840-4872. 2017. <a href="http://www.jmlr.org/papers/volume18/17-197/17-197.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
        <li>Koval I, Schiratti JB, Routier A, Bacci M, Colliot M, Allassonnière S, Durrleman S. <em>Spatiotemporal propagation of the cortical atrophy: population and individual patterns</em>. In <strong><em><u>Frontiers in Neurology</u></em></strong> 9, 2018. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5945895/" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
   </ul>
   &nbsp;
   <div class="row">
   <div class="col-sm-4">[su_button url="https://disease-progression-modelling.github.io/pages/models/disease_course_mapping.html" size="5" style="flat" background="#702082" color="white" center="yes"]Website[/su_button]</div>
   <div class="col-sm-4">[su_button url="https://gitlab.com/icm-institute/aramislab/leaspy" size="5" style="flat" background="#702082" color="white" center="yes"]Code repository[/su_button]</div>
   <div class="col-sm-4">[su_button url="mailto:stanley.durrleman(at)inria(dot)fr" size="5" style="flat" background="#702082" color="white" center="yes"]Contact[/su_button]</div>
   </div>








<h2>APPRIMAGE Study group</h2>
The aim of this project is to design and validate deep learning methods for computer-assisted diagnosis of neurological disorders using a
 very large dataset (over 100,000 patients) from the AP-HP data warehouse.

Olivier Colliot, Ninon Burgos, Simona Bottani, Sophie Loizillon<sup>1</sup>,
Didier Dormont <sup>1,2</sup>, Stéphane Lehéricy<sup>2, 21, 22</sup>, Samia Si Smail Belkacem, Sebastian Ströer<sup>2</sup>,
Nathalie Boddaert<sup>3</sup>,
Farida Benoudiba, Ghaida Nasser, Claire Ancelet, Laurent Spelle<sup>4</sup>,
Hubert Ducou-Le-Pointe<sup>5</sup>,
Catherine Adamsbaum<sup>6</sup>,
Marianne Alison<sup>7</sup>,
Emmanuel Houdart<sup>8</sup>,
Robert Carlier <sup>9,17</sup>,
Myriam Edjlali<sup>9</sup>,
Betty Marro<sup>10,11</sup>,
Lionel Arrive<sup>10</sup>,
Alain Luciani<sup>12</sup>,
Antoine Khalil<sup>13</sup>,
Elisabeth Dion<sup>14</sup>,
Laurence Rocher<sup>15</sup>,
Pierre-Yves Brillet<sup>16</sup>,
Paul Legmann, Jean-Luc Drape<sup>18</sup>,
Aurélien Maire, Stéphane Bréant, Christel Daniel, Martin Hilka, Yannick Jacob, Julien Dubiel, Cyrina Saussol, Rafael Gozlan<sup>19</sup>,
Florence Tubach, Jacques Ropers, Antoine Rozès, Camille Nevoret<sup>20</sup>.

Location: 

<sup>1</sup> Sorbonne Université, Institut du Cerveau - Paris Brain Institute, Inserm, CNRS, AP-HP, Hôpital de la Pitié Salpêtrière, Inria, Aramis project-team, F-75013, Paris, France
<sup>2</sup> AP-HP, Hôpital de la Pitié Salpêtrière, Department of Neuroradiology, F-75013, Paris, France
<sup>3</sup> AP-HP, Hôpital Necker, Department of Radiology, F-75015, Paris, France
<sup>4</sup> AP-HP, Hôpital Bicêtre, Department of Radiology, F-94270, Le Kremlin-Bicêtre, France
<sup>5</sup> AP-HP, Hôpital Armand-Trousseau, Department of Radiology, F-75012, Paris, France
<sup>6</sup> AP-HP, Hôpital Bicêtre, Department of Pediatric Radiology, F-94270, Le Kremlin-Bicêtre, France
<sup>7</sup> AP-HP, Hôpital Robert-Debré, Department of Radiology, F-75019, Paris, France
<sup>8</sup> AP-HP, Hôpital Lariboisière , Department of Neuroradiology, F-75010, Paris, France
<sup>9 </sup>AP-HP, Hôpital Raymond-Poincaré, Department of Radiology, F- 92380, Garches, France
<sup>10</sup> AP-HP, Hôpital Saint-Antoine, Department of Radiology, F-75012, Paris, France
<sup>11</sup> AP-HP, Hôpital Tenon, Department of Radiology, F-75020, Paris, France
<sup>12</sup> AP-HP, Hôpital Henri-Mondor, Department of Radiology, F-94000, Créteil, France
<sup>13 </sup>AP-HP, Hôpital Bichat, Department of Radiology, F-75018, Paris, France
<sup>14</sup> AP-HP, Hôpital Hôtel-Dieu, Department of Radiology, F-75004, Paris, France
<sup>15</sup> AP-HP, Hôpital Antoine-Béclère, Department of Radiology, F-92140, Clamart, France
<sup>16</sup> AP-HP, Hôpital Avicenne, Department of Radiology, F-93000, Bobigny, France
<sup>17</sup> AP-HP, Hôpital Ambroise Paré, Department of Radiology, F-92100 104, Boulogne-Billancourt, France
<sup>18 </sup>AP-HP, Hôpital Cochin, Department of Radiology, F-75014, Paris, France
<sup>19</sup> AP-HP, WIND department, F-75012, Paris, France
<sup>20 </sup>AP-HP, Unité de Recherche Clinique, Hôpital de la Pitié Salpêtrière, Department of Neuroradiology, F-75013, Paris, France

<sup>21</sup> Sorbonne Université, Institut du Cerveau - Paris Brain Institute, Inserm, CNRS, AP-HP, Hôpital de la Pitié Salpêtrière, F-75013, Paris, France

<sup>22</sup> ICM, Centre de NeuroImagerie de Recherche - CENIR, Paris, France', 'APPRIMAGE', '', 'inherit', 'closed', 'closed', '', '1969-autosave-v1', '', '', '2025-01-15 16:03:39', '2025-01-15 15:03:39', '', 1969, 'https://www.aramislab.fr/?p=2029', 0, 'revision', '', 0);





### key words 

'Uncategorized', 'uncategorized',
 'post-format-image', 'post-format-image',
 'Slider Home', 'slider-home',
 'News', 'news',
 'Slider Home v2', 'slider-home-v2',
 'Main menu', 'main-menu',
 'OnePage Front Page Menu', 'onepage-front-page-menu',
 'positions', 'positions',
 'publications', 'publications',
 'Publications', 'publications',
 'Frontotemporal Dementia', 'frontotemporal-dementia',
 'Amyotrophic Lateral Sclerosis', 'amyotrophic-lateral-sclerosis',
 'microRNAs', 'micrornas',
 'biomarkers', 'biomarkers',
 'Frontotemporal Dementia', 'frontotemporal-dementia',
 'Amyotrophic Lateral Sclerosis', 'amyotrophic-lateral-sclerosis',
 'MicroRNAs', 'micrornas',
 'Biomarkers', 'biomarkers',
 'Deep learning', 'deep-learning',
 'Neurology', 'neurology',
 'Medical imaging', 'medical-imaging',
 'Genomics', 'genomics'
 'Alzheimer\'s disease', 'alzheimers-disease'
 'Primary progressive aphasia', 'primary-progressive-aphasia'
 'Amyloid', 'amyloid'
 'Anosognosia', 'anosognosia'
 'Biomedical ethics', 'biomedical-ethics'
 'Clinical ethics', 'clinical-ethics'
 'Integrative medicine', 'integrative-medicine'
 'Dementia', 'dementia'
 'Diffusion of innovation', 'diffusion-of-innovation'
 'Innovation', 'innovation'
 'BCI training', 'bci-training'
 'MEG', 'meg'
 'EEG', 'eeg'
 'Multilayer networks', 'multilayer-networks'
 'Brain networks', 'brain-networks'
 'Disease Progression Modelling', 'disease-progression-modelling'
 'Forecast of cognitive decline', 'forecast-of-cognitive-decline'
 'multimodality', 'multimodality'
 'Prognostic Enrichment', 'prognostic-enrichment'



 <div class="classictext">
    The Aramis Lab brings together methodological researchers (computer scientists, applied mathematics) and medical 
    experts (neurology, medical imaging) to build numerical models of brain diseases from multimodal patient data: medical imaging, 
    clinical data and genomic data.</div>
 &nbsp;
 <div class="classictext">
    The Aramis Lab is a joint research team between <a href="http://www.cnrs.fr/index.php" target="_blank" rel="noopener noreferrer"><b>CNRS</b></a>,
     <a href="http://www.inria.fr/en/" target="_blank" rel="noopener"><b>Inria</b></a>, <a href="http://english.inserm.fr/"><b>Inserm</b></a> 
     and <a href="http://www.sorbonne-universite.fr/" target="_blank" rel="noopener noreferrer"><b>Sorbonne University</b></a> and belongs to the 
     <a href="https://institutducerveau-icm.org/en" target="_blank" rel="noopener noreferrer"><b>Paris Brain Institute</b></a> (ICM), which is a 
     neuroscience center based in the <span class="highlighttext">Pitié-Salpêtrière hospital</span> in Paris, the largest adult hospital in Europe.</div>

 <div></div>
 <div class="classictext">
    The team develops new data representations and statistical learning approaches that can integrate multiple types of data acquired in the 
    living patient, including medical imaging, clinical and genomic data. In turn, these models shall allow for a better understanding of disease 
    progression, and the development of new decision support systems for diagnosis, prognosis and design of clinical trials.</div>
 <div></div>
 <div class="classictext">
    We are financially supported by <a href="https://www.aramislab.fr/research-topics/#funding">these institutions</a>.</div>
 <div class="row">


 <h3 align="center">Main applications</h3>
[su_accordion]
[su_spoiler title="Alzheimer\'s disease" open="yes" style="fancy" icon="chevron" class="accordeon"]Clinical decision support systems, Disease progression models, Network alterations[/su_spoiler]
[su_spoiler title="Fronto-temporal dementia" style="fancy" icon="chevron" class="accordeon"]Modeling the presymptomatic phase in genetic forms of the disease[/su_spoiler]
[su_spoiler title="Multiple sclerosis" style="fancy" icon="chevron" class="accordeon"]Using multimodal neuroimaging to track disease progression[/su_spoiler]
[su_spoiler title="Parkinson\'s disease" style="fancy" icon="chevron" class="accordeon"]Modeling disease progression, Computer-aided adjustment of treatment[/su_spoiler]
[su_spoiler title="Brain computer interfaces" style="fancy" icon="chevron" class="accordeon"]Using BCI for rehabilitation, BCI systems based on network analysis[/su_spoiler]
[/su_accordion]


<h3 align="center">Key methododological domains</h3>
[su_accordion]
[su_spoiler title="Machine Learning" open="yes" style="fancy" icon="chevron" class="accordeon"][/su_spoiler]
[su_spoiler title="Medical image processing" style="fancy" icon="chevron" class="accordeon"]Magnetic resonance Imaging (MRI), Positron Emission Tomography (PET)[/su_spoiler]
[su_spoiler title="Morphometry and shape analysis" style="fancy" icon="chevron" class="accordeon"]Riemannian Geometry, Diffeomorphisms[/su_spoiler]
[su_spoiler title="Complex network theory" style="fancy" icon="chevron" class="accordeon"]Graph analysis, Multilayer networks, Network dynamics[/su_spoiler]
[su_spoiler title="Longitudinal analysis" style="fancy" icon="chevron" class="accordeon"]Non-linear mixed effects models, Riemannian geometry, Longitudinal shape models[/su_spoiler]
[/su_accordion]





<h2>Most representative publications</h2>
<h3 align="center">Networks</h3>
<ul>
 	<li>De Vico Fallani F, Richiardi J, Chavez M, Achard S. <em>Graph analysis of functional brain networks: practical issues in translational neuroscience</em>. <strong><em><u>Philosophical Transactions of the Royal Society B: Biological Sciences</u></em></strong> 369:1653, 20130521–20130521, 2014. <a href="http://arxiv.org/pdf/1406.7391.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>De Vico Fallani F, Latora V, Chavez, M. <em>A Topological Criterion for Filtering Information in Complex Brain Networks</em>. <strong><em><u>PLOS Computational Biology</u></em></strong>, 13(1), e1005305, 2017.<a href="http://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1005305&amp;type=printable" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>De Vico Fallani F, Corazzol M, Sternberg JR, Wyart C, Chavez M. <em>Hierarchy of neural organization in the embryonic spinal cord: Granger-causality graph analysis of in vivo calcium imaging data</em>. <strong><em><u>IEEE Transactions on Neural Systems and Rehabilitation Engineering</u></em></strong> 23(3):333-341, 2014. <a href="https://arxiv.org/ftp/arxiv/papers/1407/1407.4380.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Obando C, De Vico Fallani F. <em>A statistical model for brain networks inferred from large-scale electrophysiological signals</em>. <strong><em><u>Journal of the Royal Society Interface</u></em></strong> (Accepted for Publication), 2017. <a href="https://arxiv.org/pdf/1611.06893.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>De Vico Fallani F, Nicosia V, Latora V, Chavez M. <em>Non-parametric resampling of random walks for spectral networks clustering</em>. <strong><em><u>Physical Review E</u></em></strong> 89, 012802, 2013. <a href="http://arxiv.org/pdf/1304.4156.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Guillon J, Attal Y, Colliot O, La Corte V, Dubois B, Schwartz D, Chavez M, De Vico Fallani F <em>Loss of brain inter-frequency hubs in Alzheimer\'s disease. In </em><strong><em><u> Scientific Reports</u></em></strong> 7;10879. 2017.<a href="https://www.nature.com/articles/s41598-017-07846-w" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Corsi M-C, Chavez M, Schwartz D, George N, Hugueville L, Kahn A E, Dupont S, Bassett D S, De Vico Fallani F,
<em>Functional disconnection of associative cortical areas predicts performance during BCI training</em>, <strong><span style="text-decoration: underline;"><em>NeuroImage</em></span></strong>, 209: 116500, 2020. <a href="https://hal.inria.fr/hal-02438794/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
[su_divider style="default" divider_color="#702082" top="no" link_color="#702082" size="1" margin="16"]
<h3 align="center">Longitudinal</h3>
<ul>
 	<li>Schiratti J-B, Allassonniere S, Colliot O, Durrleman S.<em> Learning spatiotemporal trajectories from manifold-valued longitudinal data</em>. In <strong><em><u>Advances in Neural Information Processing Systems</u></em></strong> pp. 2395–2403, 2015. <a href="http://machinelearning.wustl.edu/mlpapers/paper_files/NIPS2015_5663.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Schiratti J-B, Allassonniere S, Colliot O, Durrleman S.<em>A Bayesian mixed-efects model to learn trajectories of changes from repeated manifold-valued observations</em>. In <strong><em><u>Journal of Machine Learning Research (JMLR)</u></em></strong> 18(1), 4840-4872. 2017. <a href="http://www.jmlr.org/papers/volume18/17-197/17-197.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Koval I, Schiratti JB, Routier A, Bacci M, Colliot M, Allassonnière S, Durrleman S. <em>Spatiotemporal propagation of the cortical atrophy: population and individual patterns</em>. In <strong><em><u>Frontiers in Neurology</u></em></strong> 9, 2018. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5945895/" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
[su_divider style="default" divider_color="#702082" top="no" link_color="#702082" size="1" margin="16"]
<h3 align="center">Morphometry</h3>
<ul>
 	<li>Durrleman S, Pennec X, Trouvé A, Braga J, Gerig G, Ayache N. <em>Toward a comprehensive framework for the spatiotemporal statistical analysis of longitudinal shape data</em>. <strong><em><u>International Journal of Computer Vision</u></em></strong> 103(1):22-59, 2013. <a href="https://who.rocq.inria.fr/Stanley.Durrleman/papers/Durrleman_IJCV_2012_Longitudinal.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Durrleman S, Prastawa M, Charon N, Korenberg JR, Joshi S, Gerig G, Trouvé A. <em>Morphometry of anatomical shape complexes with dense deformations and sparse parameters</em>. <strong><em><u>NeuroImage</u></em></strong> 101:35–49, 2014. <a href="https://who.rocq.inria.fr/Stanley.Durrleman/papers/2013_Neuroimage_Durrleman_author_copy.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Gori P, Colliot O, Marrakchi-Kacem L, Worbe Y, Poupon, C, Hartmann A, Ayache N, Durrleman S. <em>A Bayesian framework for joint morphometry of surface and curve meshes in multi-object complexes</em>. <strong><em><u>Medical Image Analysis</u></em></strong> , 35, 458-474, 2017.<a href="https://hal.inria.fr/hal-01359423/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Louis M, Charlier B, Jusselin P, Susovan P, Durrleman S. <em>A fanning scheme for the parallel transport along geodesicson Riemmanian manifolds</em>. In <strong><em><u>SIAM journal on Numerical Analysis</u></em></strong> 2017. 56(4), 256-2584<a href="https://hal.archives-ouvertes.fr/hal-01560787/file/JacobisFan.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Bone A, Colliot O, Durrleman S<em>Learning distributions of shape trajectories from longitudinal datasets: a hierarchical model on a manifold of diffeomorphisms</em>. In <strong><em><u>Computer Vision and Pattern Recognition (CVPR)</u></em></strong> 9271-9280, 2018. <a href="http://openaccess.thecvf.com/content_cvpr_2018/papers/Bone_Learning_Distributions_of_CVPR_2018_paper.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
[su_divider style="default" divider_color="#702082" top="no" link_color="#702082" size="1" margin="16"]
<h3 align="center">Machine Learning</h3>
<ul>
 	<li>Cuingnet R, Glaunès JA, Chupin M, Benali H, Colliot O. The ADNI. <em>Spatial and anatomical regularization of SVM: a general framework for neuroimaging data</em>. <strong><em><u>IEEE Transactions on Pattern Analysis and Machine Intelligence</u></em></strong>, 35(3):682-696, 2013. <a href="../files/papers/ieee_tpami_2013_published.pdf" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Samper-González J, Burgos N, Bottani S, Fontanella S, Lu P, Marcoux A, Routier A, Guillon J, Bacci M, Wen J, Bertrand A, Bertin H, Habert MO, Durrleman S, Evgeniou T, Colliot O. <em>Reproducible evaluation of classification methods in Alzheimer\'s disease: Framework and application to MRI and PET data</em>. <strong><em><u>Neuroimage</u></em></strong><em>, </em>183: 504–521, 2018. <a href="https://hal.inria.fr/hal-01858384/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Wen J, Thibeau-Sutre E, Samper-González J, Routier A, Bottani S, Durrleman S, Burgos N, Colliot O: <em>Convolutional Neural Networks for Classification of Alzheimer’s Disease: Overview and Reproducible Evaluation</em>, <em><span style="text-decoration: underline;"><strong>Medical Image Analysis</strong></span></em>, 63: 101694, 2020 <a href="https://hal.inria.fr/hal-02562504/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Ansart A, Epelbaum S, Gagliardi G, Colliot O, Dormont D, Dubois B, Hampel H, Durrleman S. <em>Prediction of amyloidosis from neuropsychological and MRI data for cost effective inclusion of pre-symptomatic subjects in clinical trials</em>. In <span style="text-decoration: underline;"><strong><em>Multimodal Learning for Clinical Decision Support</em></strong></span>, 357-364. 2017. <a href="https://hal.inria.fr/hal-01578422/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Wei, W., Poirion, E., Bodini, B., Durrleman, S., Ayache, N., Stankoff, B., Colliot, O. <em>Predicting PET-derived Demyelination from Multimodal MRI using Sketcher-Refiner Adversarial Training for Multiple Sclerosis</em>, <em><span style="text-decoration: underline;"><strong>Medical Image Analysis</strong></span></em>, 58: 101546, 2019. <a href="https://hal.inria.fr/hal-02276634/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
[su_divider style="default" divider_color="#702082" top="no" link_color="#702082" size="1" margin="16"]
<h3 align="center">Clinical studies</h3>
<ul>
 	<li>Dubois B, Chupin M, Hampel H, Lista S, Cavedo E, Croisile B, Tisserand GL, Touchon J, Bonafe A, Ousset PJ, Ait Ameur A, Rouaud O, Ricolfi F, Vighetto A, Pasquier F, Delmaire C, Ceccaldi M, Girard N, Lehéricy S, Tonelli I, Duveau F, Colliot O, Garnero L, Sarazin M, Dormont D. <em>Donepezil decreases annual rate of hippocampal atrophy in suspected prodromal Alzheimer’s disease</em>. <strong><em><u>Alzheimer’s &amp; Dementia</u></em></strong> 11(9):1041–1049, 2015. <a href="http://ac.els-cdn.com/S1552526014028568/1-s2.0-S1552526014028568-main.pdf?_tid=c29fa44a-fe95-11e6-ab60-00000aacb361&amp;acdnat=1488383165_2b9d236106e5e88c1f1c03d8f47840d0" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Hamelin L, Bertoux M, Bottlaender M, Corne H, Lagarde J, Hahn V, Mangin JF, Dubois B, Chupin M, Cruz de Souza L, Colliot O, Sarazin M, S<em>ulcal morphology as a new imaging marker for the diagnosis of early onset Alzheimer\'s disease</em>, <strong><em><u>Neurobiology of Aging</u></em></strong>, 36(11):2932-9, 2015 <a href="http://ac.els-cdn.com/S0197458015003759/1-s2.0-S0197458015003759-main.pdf?_tid=942c296c-fe95-11e6-9aee-00000aab0f6b&amp;acdnat=1488383087_72117df68c119ad1103e3229515aa491"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Betrand A, Wen J, Rinaldi D, Houot M, Sayah S, Camuzat A, Fournier C, Fontanella S, Routier A, Couratier P, Pasquier F, Habert MO, Hannequin D, Martinaud O, Caroppo P, Levy R, Dubois B, Brice A, Durrleman S, Colliot O. <em>Early Cognitive, Structural, and Microstructural Changes in Presymptomatic C9orf72 Carriers Younger Than 40 Years</em>. In <strong><em><u>JAMA neurology</u></em></strong> 75(2);236-245, 2018. <a href="https://hal.inria.fr/hal-01654000/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Dubois B, Epelbaum S, Nyasse F, Bakardjian H, Gagliardi G, Uspenskaya O, Houot M, Lista S, Cacciamani F, Potier MC, Bertrand A, Lamari F, Benali H, Mangin JF, Colliot O, Genthon R, Habert MO, Hampel H; INSIGHT-preAD study group. <em> Cognitive and neuroimaging features and brain β-amyloidosis in individuals at risk of Alzheimer\'s disease (INSIGHT-preAD): a longitudinal observational study.</em>. In <strong><em><u></u>Lancet Neurol.</em></strong>. 2018, Apr;17(4):335-346. <a href="https://hal.sorbonne-universite.fr/hal-01777834/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
 	<li>Wen J, Zhang H, Alexander DC, Durrleman S, Routier A, Rinaldi D, Jouot M, Couratier P, Hannequin D, Pasquier F, Zhang J, Colliot O, Le Ber I, Bertrand A. <em>Neurite density is reduced in the presymptomatic phase of C9orf72 disease</em>. In <strong><em><u>J Neurol Neurosurg Psychiatry</u></em></strong> 318994. 2018. <a href="https://hal.inria.fr/hal-01907482/document" target="_blank" rel="noopener noreferrer"><img src="../files/images/pdf-logo_crop2.jpg" alt="PDF" width="20" height="20" align="absmiddle" border="0" /> Paper in PDF</a></li>
</ul>
&nbsp;

 
 <h2>Full list of publications</h2>
Here is a <a href="https://haltools.inria.fr/Public/afficheRequetePubli.php?labos_exp=aramis+%3B+nemesis-cricm&amp;CB_auteur=oui&amp;CB_titre=oui&amp;CB_article=oui&amp;langue=Francais&amp;tri_exp=annee_publi&amp;tri_exp2=typdoc&amp;tri_exp3=date_publi&amp;ordre_aff=TA&amp;Fen=Aff&amp;css=../css/VisuCondense.css" target="_blank" rel="noopener noreferrer">link</a> to our publications on the open archive HAL.
