<!DOCTYPE HTML>
<!--
	TXT by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research topics</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="../assets/css/main.css" />
	</head>
	<body class="homepage is-preload">
		<div id="page-wrapper">
			<!-- Nav -->
			<nav id="nav">
				<ul>
					<li class ="logo_aramis"><a href="../index.html"><img src="../images/icons/aramis/icon_ARAMISLAB_rvb.png" alt="" /></a></li>
					<li class ="current"><a href="../index.html">Home</a></li>
					<li>
						<a href="#">Research</a>
						<ul>
							<li><a href="../research/topics.html">Topics</a></li>
							<li><a href="../research/publications.html">Publications</a></li>
							<li><a href="../research/collaborations.html">Collaborations</a></li>
						</ul>
					</li>
					<li>
						<a href="#">Softwares</a>
						<ul>
							<li><a href="../softwares/clinica.html">Clinica Environment</a></li>
							<li><a href="../softwares/deformetrica.html">Deformetrica</a></li>
							<li><a href="../softwares/leaspy.html">Leaspy</a></li>
						</ul>
					</li>
					<li>
						<a href="#">The team</a>
						<ul>
							<li><a href="../the_team/people.html">People</a></li>
							<li><a href="../the_team/job_offers.html">Join us</a></li>
						</ul>
					</li>
				</ul>
			</nav>
			<!-- Main -->
				<section id="main">
					<div class="container">
						<div class="row gtr-200">
							

							<div class="col-12">
								<!-- Features -->
									<section class="box features">
										<h2 class="major"><span>New representations from multimodal medical images</span></h2>
										<div>
											<div class="row">
												Combining multiple neuroimaging modalities is necessary to obtain a comprehensive picture 
												of alterations in brain diseases (atrophy anatomical disconnections functional connectivity 
												alterations metabolic alterations abnormal protein depositsâ€¦). Such a combination is a 
												non-trivial task because different types of information are conveyed by the different 
												modalities which in turns leads to different natural data representations (meshes and 
												curves for geometrical information signals networks). We propose to build new integrated 
												data representations from multiple modalities. Such representations will be subsequently 
												entered into statistical models and decision support systems. Specifically we will introduce 
												representations that can integrate geometrical information (anatomical surfaces extracted 
												from anatomical MRI white matter tracts extracted from diffusion MRI) together with functional 
												(PET ASL EEG/MEG) and microstructural information.

											</div>
										</div>
									</section>
							</div>

							<div class="col-12">
								<!-- Features -->
									<section class="box features">
										<h2 class="major"><span>Network theoretic approaches to integrate heterogeneous brain networks</span></h2>
										<div>
											<div class="row">
												The complexity of biological systems often emerges from interactions between 
												components at multiple spatial and temporal scales. Neglecting this information and 
												analyzing separately the levels of such scales is an oversimplification of the real 
												phenomenon. We propose a methodological framework that aims on the one hand to integrate 
												information from networks describing different modes of connectivity (e.g. multi-modal
												cross-frequency) and on the other hand to statistically model the organizational 
												mechanisms of temporally dynamic networks (e.g. nonstationary longitudinal). Target 
												applications include: 
												<ul>
													    <li>human learning in brain-computer interface </li>
                                                        <li>prediction of neurodegenerative disease progression and </li>
                                                        <li>identification of driving nodes in biological pathways (gene expression networks) </li>
                                                    </ul>
												</ul>
											</div>

											<div class="row">
												<h2>References</h2>
												<ul>
													<li>Network neuroscience for optimizing brain-computer interfaces. De Vico Fallani et al. 2018. (Paper)[https://arxiv.org/pdf/1807.05616.pdf] </li>
													<li>A Topological Criterion for Filtering Information in Complex Brain Networks. De Vico Fallani et al. 2017. (Paper)[https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005305]</li>
													<li>Graph analysis of functional brain networks: practical issues in translational neuroscience. De Vico Fallani. 2014. (Paper)[http://rstb.royalsocietypublishing.org/content/369/1653/20130521.short]</li>
												</ul>
												</ul>
											</div>
											
										</div>
									</section>
							</div>
							
							<div class="col-12">
								<!-- Features -->
									<section class="box features">
										<h2 class="major"><span>Spatio-temporal models to build trajectories of disease progression from longitudinal data</span></h2>
										<div>
											<div class="row">
												Longitudinal data sets are collected to capture variable temporal phenomena which may be due 
												to ageing or disease progression for instance. They consist in the observation of several 
												individuals each of them being observed at multiple points in time. The statistical 
												exploitation of such data sets is notably difficult since data of each individual follow 
												a different trajectory of changes and at its own pace. This difficulty is further increased 
												if observations take the form of structured data like images or measurements distributed 
												at the nodes of a mesh and if the measurements themselves are normalized data or positive 
												definite matrices for which usual linear operations are not defined. Our team has contributed 
												to the definition of a generic theoretical and algorithmic framework for learning typical 
												trajectories from longitudinal data sets. This framework is built on tools from the Riemannian 
												geometry to describe trajectories of changes for any kind of data and their variability within 
												a group both in terms of the direction of the trajectories and the pace at which trajectories 
												are followed. The inference is based on a stochastic Expectation Maximization (EM) algorithm 
												coupled with Markov Chain Monte Carlo methods.
											</div>
										</div>
									</section>
							</div>

							<div class="col-12">
								<!-- Features -->
									<section class="box features">
										<h2 class="major"><span>Decision support systems for diagnosis prognosis and design of clinical trials</span></h2>
										<div>
											<div class="row">
												Based on the new representations and statistical models we design decision support systems for diagnosis 
												prognosis and design of clinical trials. These systems are based on: 
												<ul>
													    <li>personalization of statistical models to predict evolution at the patient level; </li>
                                                        <li>new machine learning approches for classification and regression on high-dimensional and structured data; </li>
                                                        <li>content-based retrieval techniques. </li>
                                                        <li>the use of genetic algorithms for optimization </li>
                                                    </ul>
												</ul>
											</div>
										</div>
									</section>
							</div>
						</div>
					</div>
				</section>

				<!-- Footer -->
				<footer id="footer">
					<div class="container-footer">
						<div class="row gtr-50">
							<div class="col-12">

								<!-- Contact -->
									<section>
										<h2 class="major"><span>Get in touch</span></h2>
										<ul class="contact">
											<li><a class="icon brands fa-twitter" href="https://x.com/AramisLabParis"><span class="label">Twitter</span></a></li>
											<li><a class="icon fas fa-location-arrow" href="https://www.google.com/maps/place/Institut+du+Cerveau+-+Paris+Brain+Institute/@48.8351807,2.3641893,619m/data=!3m2!1e3!5s0x47e6718ab3c2c389:0xe5631a1aac15ece4!4m14!1m7!3m6!1s0x47e671f4647492ff:0x7700b00040eb24c6!2sInstitut+du+Cerveau+-+Paris+Brain+Institute!8m2!3d48.8351807!4d2.3641893!16s%2Fg%2F1tycvzk8!3m5!1s0x47e671f4647492ff:0x7700b00040eb24c6!8m2!3d48.8351807!4d2.3641893!16s%2Fg%2F1tycvzk8?entry=ttu&g_ep=EgoyMDI1MDExNS4wIKXMDSoASAFQAw%3D%3D"><span class="label">Location</span></a></li>
											<li><a class="icon far fa-envelope" href="mailto:olivier.colliot@cnrs.fr"><span class="label">Mail</span></a></li>
											<li><a class="icon brands fa-linkedin" href="https://www.linkedin.com/company/institutducerveau/"><span class="label">LinkedIn</span></a></li>
										</ul>
									</section>

							</div>
						</div>

						<!-- Copyright -->
							<!-- <div id="copyright">
								<ul class="menu">
									<li>&copy; Untitled. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
								</ul>
							</div> -->

					</div>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.dropotron.min.js"></script>
			<script src="../assets/js/jquery.scrolly.min.js"></script>
			<script src="../assets/js/browser.min.js"></script>
			<script src="../assets/js/breakpoints.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<script src="../assets/js/main.js"></script>

	</body>
</html>